server/models/User.js

```diff
@@ -5,16 +5,20 @@ module.exports = (sequelize, DataTypes) => {
             allowNull: false,
             unique: true
         },
-        passwordHash: {
-            type: DataTypes.STRING,
-            allowNull: false
-        },
-        preferredName: DataTypes.STRING,
+        // preferredName: DataTypes.STRING,
```

Comment:
why are we leaving this commented

Reply:
I think Rutwa mentioned she didn't want people to give themselves a name as it could be inappropriate etc..